test/gemojione_test.rb

```diff
@@ -89,6 +89,36 @@
       assert_equal "I <img alt=\"❤\" class=\"emoji\" src=\"http://localhost:3000/2764.png\"> Emoji", replaced_string
     end
 
+    it 'should replace unicode moji with span tag for spritesheet' do
+      with_emoji_config(:use_sprite, true) do
+        base_string = "I ❤ Emoji"
+        replaced_string = Gemojione.replace_unicode_moji_with_images(base_string)
+        assert_equal "I <span class=\"emojione-2764\" alt=\"❤\" title=\"❤\">⃣❤</span> Emoji", replaced_string
+      end
+    end
+
+    it 'should replace short name moji with span tag for spritesheet' do
+      with_emoji_config(:use_sprite, true) do
+        base_string = "I :heart: Emoji"
+        replaced_string = Gemojione.replace_named_moji_with_images(base_string)
+        assert_equal "I <span class=\"emojione-2764\" alt=\"❤\" title=\"❤\">⃣❤</span> Emoji", replaced_string
+      end
+    end
+
+    it 'should replace ascii with image tag for' do
+      base_string = "Hello Emoji =D"
+      replaced_string = Gemojione.replace_ascii_moji_with_images(base_string)
+      assert_equal "Hello Emoji <img alt=\"😃\" class=\"emoji\" src=\"http://localhost:3000/1F603.png\">", replaced_string
+    end
+
+    it 'should replace ascii with span tag for spritesheet' do
+      with_emoji_config(:use_sprite, true) do
+        base_string = "Hello Emoji =D"
+        replaced_string = Gemojione.replace_ascii_moji_with_images(base_string)
+        assert_equal "Hello Emoji <span class=\"emojione-1F603\" alt=\"😃\" title=\"😃\">⃣😃</span>", replaced_string
+      end
+    end
+
```

Comment:
~~These 2 first tests should test the new methods (`replace_unicode_moji_with_images` and `replace_named_moji_with_images` respectively) without the use of spritesheet option, like the 3rd one. That should be tested separately because it's a different and independent feature.~~

Sorry my bad, the first one is all right. The second one, `it 'should replace short name moji with span tag for spritesheet' do` should be under `describe 'replace_named_moji_with_images' do` since it belongs to that method.

The third and fourth ones should be under a new `describe` since it's a new method